ansible/roles/openstack/init_vm/vars/CentOS-7.yml

```diff
@@ -0,0 +1,5 @@
+openstack_packages:
```

Comment:
Why do we this centos-7.yml file ?

Reply:
To specify the packages that are used in CentOS 7. It was the default until now, but will stop being from now on.

Reply:
If it is not anymore the default, should it be then removed ?

Reply:
I'll reorganize the files.

Reply:
The default  (`main.yml`) has now the configuration for CentOS 8.